test/externalTests/bleeps.sh

```diff
@@ -46,8 +46,8 @@ function bleeps_test
     local compile_only_presets=()
     local settings_presets=(
         "${compile_only_presets[@]}"
-        #ir-no-optimize            # Compilation fails with: "YulException: Variable param_0 is 2 slot(s) too deep inside the stack."
-        #ir-optimize-evm-only      # Compilation fails with: "YulException: Variable param_0 is 2 slot(s) too deep inside the stack."
+        #ir-no-optimize            # Compilation fails with: "YulException: Variable expr_15509_mpos is 4 too deep in the stack". No memoryguard was present.
+        #ir-optimize-evm-only      # Compilation fails with: "YulException: Variable expr_15260_mpos is 4 too deep in the stack". No memoryguard was present.
         ir-optimize-evm+yul
         #legacy-no-optimize        # Compilation fails with: "CompilerError: Stack too deep, try removing local variables."
         #legacy-optimize-evm-only  # Compilation fails with: "CompilerError: Stack too deep, try removing local variables."
```

Comment:
These two would actually start working if we enabled the new steps also for legacy.